media/CMakeLists.txt

```diff
@@ -1,18 +1,12 @@
 
+add_subdirectory(drivers)
```

Comment:
This change is unrelated to the purpose of this PR and does not make much sense. The configuration files in drivers/etc are not related to sound events at all. Please revert it (and remove the corresponding CMake file you've added).